### PR TITLE
1937 angular implement slide toggle tests utils

### PR DESCRIPTION
**Pull request by Sudalius**

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java  1
- jdi-light-angular/src/test/java/io/github/epam/unit/TestsInit.java  1
- jdi-light-angular/src/test/java/io/github/epam/testng/TestNGListener.java  3
- jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/SlideToggleTests.java  1
- jdi-light-angular/src/main/java/elements/common/Checkbox.java  2
- jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java  1
- jdi-light-angular/src/main/java/elements/common/SlideToggle.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599257&bid=18343552)

File: jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+public class CheckboxTest {
+
+    private static final String ANGULAR_SELECTED_CLASS = "mat-checkbox-checked";
+    private static final String ANGULAR_DISABLE_CLASS = "mat-checkbox-disabled";
+
+    @InjectMocks
+    private Checkbox checkbox;
+
+    @Mock
+    private UIElement uiElement;
```
Comment (codacy-review) [inline jdi-light-angular/src/test/java/io/github/epam/unit/tests/CheckboxTest.java:24]: Codacy Issue found: [Avoid unused private fields such as 'uiElement'.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599257)

File: jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+public class SlideToggleTest {
+
+    private static final String ANGULAR_SELECTED_CLASS = "mat-checked";
+    private static final String ANGULAR_DISABLE_CLASS = "mat-disabled";
+
+    @InjectMocks
+    private SlideToggle slideToggle;
+
+    @Mock
+    private UIElement uiElement;
```
Comment (codacy-review) [inline jdi-light-angular/src/test/java/io/github/epam/unit/tests/SlideToggleTest.java:26]: Codacy Issue found: [Avoid unused private fields such as 'uiElement'.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5599257)

**Comment by AkioFaris** [comment]
This PR is declined (1937-Angular-Implement-Slide-Toggle-tests-utils branch is temporary)

**Pull request closed without merging**